### PR TITLE
Adding Episode #31 of Empower Apps

### PR DESCRIPTION
**Pull request by leogdion**

### CODE REVIEW

**Comment by pedrommcarrasco** [comment]
Great episode @leogdion! 👍
However, for now we only share episodes from "The Swift Community Podcast", "Swift by Sundell" and "Stacktrace". There're a lot of good podcasts around but we can't just spam our users with all of them.
We'll open new slots for 1 or 2 podcasts and newsletters soon.

**Comment by leogdion** [comment]
Thanks @pedrommcarrasco 
I love the app and want to contribute in some way (besides my own content). Is there anything else you are looking for exactly? Any other way I can help?
We talked about the app on our episode and I'm really liking it.

**Comment by pedrommcarrasco** [comment]
Oh, I didn't even noticed it while listening 😱
Maybe that makes it worth an exception 🤔 I'll think about it and let you know later today!

When it comes to contributing, right now, since the app itself isn't open-sourced, you can help with the most important thing: **content**.

Basically, if you spot news, events or articles that might be worth sharing, open an issue or a PR. For example, for this week's articles we have a new issue open: https://github.com/pedrommcarrasco/CocoaHub.content/issues/35

Let me know if you have any questions!

**Comment by pedrommcarrasco** [comment]
Holy, how did I miss this, it's right at the start 🤦‍♂️
Thanks @leogdion

**Comment by leogdion** [comment]
Thanks 

PS: Ep.31 at 3:30... hopefully I pronounced your name correctly 😁

**Comment by pedrommcarrasco** [comment]
You did, best try so far 😂

**Comment by pedrommcarrasco** [comment]
After thinking about it for a while, I think we'll have a new rule:
- We'll also share episodes outside of our "whitelist" if they mention CocoaHub (mentioning it in every single episode doesn't count of course 😂)

File: News/341-empowerAppsShowPodcast031.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "EmpowerApps Show #31: “Future Proofing Your Apps with Gui Rambo”",
+    "description": "In this episode, Leo talks with Gui Rambo about how current apps could be built while taking into account possible future hardware from Apple.",
```
Comment (pedrommcarrasco) [inline News/341-empowerAppsShowPodcast031.json]: Can you update your name to `Leo Dion` here? Also, can you mention you mentioned CocoaHub?

Comment (leogdion) [reply News/341-empowerAppsShowPodcast031.json]: I updated the json file accordingly.

Comment (pedrommcarrasco) [reply News/341-empowerAppsShowPodcast031.json]: What do you think of? 
`In this episode, Leo Dion discusses with Gui Rambo about how current apps could be built while taking into account possible future hardware from Apple. They also talk about apps for developers such as Gui's WWDC App, Apple's Developer App and CocoaHub.`

Comment (leogdion) [reply News/341-empowerAppsShowPodcast031.json]: Great!

**Comment by pedrommcarrasco** [comment]
Thank you for contributing @leogdion 👍

**Comment by leogdion** [comment]
Thanks for adding it. Do you want me to wait to add future episodes or can I go ahead and do that?

**Comment by pedrommcarrasco** [comment]
Let's wait until we open new slots for podcasts and newsletters. This episode was an exception because you promoted CocoaHub, so I think it's fair to promote your work too.

**Pull request merged**